app/views/articles/show.html.erb

```diff
@@ -9,3 +9,31 @@
                     turbo_confirm: "Are you sure?"
                   } %></li>
 </ul>
+
+<h2>Comments</h2>
+<% @article.comments.each do |comment| %>
+  <p>
+    <strong>Commenter</strong>
+    <%= comment.commenter%>
+  </p>
+
+  <p>
+    <strong>Comment</strong>
+    <%= comment.body%>
+  </p>
+<% end %>
+
+<h2>Add a comment:</h2>
+<%= form_with model: [ @article, @article.comments.build ] do |form| %>

```

Comment:
What does `form_width` do?

Reply:
This is a helper method that helps in the view. This method permits to build of a basic form of HTML.